androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/PlayerControls.kt

```diff
@@ -42,6 +43,14 @@ fun PlayerControls(
     customization: ControlsCustomization,
     settingsControlsCustomization: SettingsControlsCustomization
 ) {
+    val onPreviousClickState by rememberUpdatedState(onPreviousClick)
+    val onNextClickState by rememberUpdatedState(onNextClick)
+    val onPauseToggleState by rememberUpdatedState(onPauseToggle)
+    val onSpeedSelectedState by rememberUpdatedState(onSpeedSelected)
+    val onIsLoopEnabledSelectedState by rememberUpdatedState(onIsLoopEnabledSelected)
+    val onSeekBarValueChangeState by rememberUpdatedState(onSeekBarValueChange)
+    val onFullScreenToggleState by rememberUpdatedState(onFullScreenToggle)
```

Comment:
Why are you using these `rememberUpdatedState`? I didn't get the meaning after reading the [documentation](https://developer.android.com/jetpack/compose/side-effects?#rememberupdatedstate)